web/src/lib/i18n/en.json

```diff
@@ -127,13 +127,14 @@
     "map_enable_description": "Enable map features",
     "map_gps_settings": "Map & GPS Settings",
     "map_gps_settings_description": "Manage Map & GPS (Reverse Geocoding) Settings",
+    "map_implications": "Enabling the map implies clients (web and mobile apps) sending requests to a third-party map server in order to display the map. These requests include the client's IP address, the Immich instance URL, as well as the coordinates and zoom level of the requested map tiles. The default tile provider is tiles.immich.cloud",
     "map_light_style": "Light style",
     "map_manage_reverse_geocoding_settings": "Manage <link>Reverse Geocoding</link> settings",
     "map_reverse_geocoding": "Reverse Geocoding",
     "map_reverse_geocoding_enable_description": "Enable reverse geocoding",
     "map_reverse_geocoding_settings": "Reverse Geocoding Settings",
     "map_settings": "Map Settings",
-    "map_settings_description": "Manage map settings",
+    "map_settings_description": "Manage map settings. <link>Alternative tile providers can be chosen by changing the style.json map theme below.</link>",
```

Comment:
Both of these links go to the same place. I'm also not a huge fan of having the entire row be a clickable button, and then also having a link embedded it in . Can we move this note and associated link to the top of the "Map & GPS Settings" section instead? (no more links in accordion descriptions).

![image](https://github.com/user-attachments/assets/bcf5fb5d-09c2-45d6-bda7-435fd645c3f1)

Something like:

> For more information about these settings, refer to the <link>documentation</link>.

That could be something like `admin.refer_to_docs` and we could link it to reverse geocoding in this case.

Reply:
I removed them all and links can be added on the web once the explanations are added in the doc.